pkg/minikube/cluster/commands.go

```diff
@@ -41,5 +41,9 @@ func GetStartCommand() string {
 		}
 	}
 	flags := strings.Join(flagVals, " ")
-	return fmt.Sprintf(startCommandFmtStr, flags, constants.RemoteLocalKubeErrPath, constants.RemoteLocalKubeOutPath)
+	return fmt.Sprintf(startCommandFmtStr, flags, constants.RemoteLocalKubeErrPath, constants.RemoteLocalKubeOutPath, constants.LocalkubePIDPath)
 }
+
+var localkubeStatusCommand = fmt.Sprintf(`
```

Comment:
Would it work to check the return code of "ps $(cat %s)"? I think that will return 0 if the process exists and 1 otherwise.

So something like:

```
if ps $(cat /var/run/docker.pid) 2>&1 1>/dev/null; then
  echo "running"
else
  echo "stopped"
fi
```